src/main/java/org/digma/intellij/plugin/assets/AssetsService.java

```diff
@@ -68,6 +69,7 @@ public String getAssets() {
 
     public void showAsset(String spanId) {
         Log.log(logger::debug, project, "showAsset called");
+        ActivityMonitor.getInstance(project).registerSpanLinkClicked("Main", "Asset");
```

Comment:
is it a single asset or assets view?

if its single 
```suggestion
        ActivityMonitor.getInstance(project).registerSpanLinkClicked("Assets", "Asset");
```

if its the click on panel then

```suggestion
        ActivityMonitor.getInstance(project).registerSpanLinkClicked("Main", "Assets");
```

Reply:
fixed